libraries/DataFlash/DataFlash.h

```diff
@@ -324,6 +324,8 @@ class DataFlash_Class
     // start page of log data
     uint16_t _log_data_page;
 
+    int _log_sending_chan = -1;
```

Comment:
Just noticed this now, should it be a `mavlink_channel_t`? We don't usually put *int*s in code.